grails-app/services/org/pih/warehouse/product/ProductService.groovy

```diff
@@ -1273,6 +1273,9 @@ class ProductService {
                            }
                         }
                         ilike("description", "%" + term)
+                        ilike("brandName", term)
+                        ilike("manufacturerCode", term)
+                        ilike("vendorCode", term)
                         ilike("upc", term)
```

Comment:
I see you adding in the `searchProductsDtos`:
```groovy
or lower(product_supplier.brand_name) like '${it}%'
```
but I don't see you adding the `brand_name` condition to `productSupplliers` in this method.
Is it intentional?